src/Webform.js

```diff
@@ -1266,6 +1274,10 @@ export default class Webform extends NestedDataComponent {
       if ('details' in error) {
         error = error.details;
       }
+      this.serverErrors = error.map((err) => {
+        err.fromServer = true;
```

Comment:
why are you setting ALL the errors to be fromServer here?  How do we know they are from the server?